benchmark/benchmark.dart

```diff
@@ -56,8 +56,10 @@ String _loadFile(String name) {
 }
 
 void _printResult(String label, double time) {
-  print('$label: ${_padLeft(time.toStringAsFixed(2), 4)}ms '
-      "${'=' * ((time * 20).toInt())}");
+  print(
+    '$label: ${_padLeft(time.toStringAsFixed(2), 4)}ms '
+    "${'=' * ((time * 20).toInt())}",
```

Comment:
I am not a fan of this lint rule. For cases like this.

I am a fan of adding trailing commas to argument lists when there is more than one argument, and they span more than 1 line without the comma.

This change here is harder for me to read though.